### PR TITLE
🐛🐛🐛 fix bug of conceptual layout

### PR DESCRIPTION
**Pull request by OsmondJiang**
accidentally find a bug when debugging sql-docs-pr repo, layout is added as global metdata in sql-docs-pr 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6061)

### CODE REVIEW

**Comment by OsmondJiang** [comment]
image

File: src/docfx/validation/MetadataValidator.cs [hunk 1]
```diff
@@ -76,7 +76,8 @@ public List<Error> ValidateMetadata(JObject metadata, FilePath filePath)
             foreach (var schemaValidator in _schemaValidators)
             {
                 // Only validate conceptual files
-                if (contentType == ContentType.Page && mime == "Conceptual" && !metadata.ContainsKey("layout"))
+                if (contentType == ContentType.Page && mime == "Conceptual" &&
+                    (!metadata.ContainsKey("layout") || string.Equals(metadata.Value<string>(), "conceptual", StringComparison.OrdinalIgnoreCase)))
```
Comment (yufeih) [inline src/docfx/validation/MetadataValidator.cs]: `Value<string>` will throw if the value is not a string?

Comment (OsmondJiang) [reply src/docfx/validation/MetadataValidator.cs]: good catch!

**Comment by OsmondJiang** [comment]
These diff are expected, take Dynamics-365-Operations as example, there is a global metadata in docfx.jon "layout: Conceptual". So the diff are caused by these markdown file finally getting validated. 😎

**Pull request merged**